### DISCUSSION TITLE
Light logo in dark theme

### DISCUSSION THREAD

**Discussion by duartecancela** [timlrx/tailwind-nextjs-starter-blog#475] [Q&A] [answered]
Hi, it's possible to change the logo image in dark mode? i would like to use a white svg image.

**Comment by dlarroder** [comment, marked as answer]
Yes, you can use the `useTheme()` hook to determine the current theme of your application. With that, you can dynamically change the logo you are rendering. For example:

````js
import React from 'react'
import { useTheme } from 'next-themes'
import LightLogo from './LightLogo'
import DarkLogo from './DarkLogo'

export default function Sample() {
  const { theme } = useTheme()

  return (
    <div>
      {theme === 'light' ? <LightLogo /> : <DarkLogo />}
      <span>Nav Menus</span>
    </div>
  )
}

````

**Reply by duartecancela** [reply]
Thank you very much, this was exactly what I was looking for :)
